### ISSUE TITLE
check for valid PS IDs

### ISSUE THREAD

**Issue by cmungall** [monarch-initiative/mondo#2339]
e.g

```
id: MONDO:0000049
name: invasive pneumococcal disease, recurrent isolated
xref: DC:0000227 {source="MONDO:equivalentTo"}
xref: OMIMPS:610799 {source="MONDO:equivalentTo", source="DC:0000227"}
xref: UMLS:CN228622 {source="MEDGEN:kboom-pr97-c99", source="MONDO:equivalentTo"}
is_a: MONDO:0020573 {source="MONDO:cjm"} ! inherited disease susceptibility
```

https://www.omim.org/phenotypicSeries/PS610799

Says invalid

You can see what is going in here by looking at the prototype entry:
https://www.omim.org/entry/610799

moved to https://www.omim.org/entry/607676

which has this PS

https://www.omim.org/phenotypicSeries/PS300755

(different than original)

**Comment by nicolevasilevsky** [comment]
Got it.

@cmungall In this example the children should be merged with the Mondo term that is equiv to the OMIM term that OMIM merged with, right?

**Comment by nicolevasilevsky** [comment]
(Maybe you were implying this, just wanted to clarify!)

**Comment by nicolevasilevsky** [comment]
@cmungall - Another question - do you want me to try to check all the existing PS?
Or is this something @matentzn could do - to check to see if any of the OMIMPS terms have been obsoleted in OMIM? (not sure if there is an easy way to do that)

**Comment by cmungall** [comment]
please don't manually look up all of these! we can make a report of all equiv xrefs that point to an obsolete or removed record

**Comment by nicolevasilevsky** [comment]
@matentzn - I am reassigning this to you - can you create the report that Chris mentioned above?
Thank you!

**Comment by matentzn** [comment]
Yes, no problem. Please add ticket to QC board and sort into the appropriate spot of the "next" column. Generally it would be good if you could do that whenever you make a new ticket - this ensures that I will do the most important ticket next.

**Comment by joeflack4** [comment]
This looks to be down my alley. Sounds like RDFLib might be good to use for this. With a returned failure, I can also easily print out all the terms that failed.

**Questions**
@cmungall @matentzn 
1. When you say 'terms', I should be looking up by CURIE, is that correct?
2. Is this not the kind of thing that could be automated as part of the build process for the OMIM ingest pipeline repo?
3. If (2) is "no", then do you want the script placed here?: https://github.com/monarch-initiative/mondo/tree/master/src/scripts
